manifest_prod.yml

```diff
@@ -1,12 +1,16 @@
 ---
 memory: 512M
 stack: lucid64
+instances: 2
+env:
+  NEW_RELIC_CONFIG_FILE: newrelic.ini
+  NEW_RELIC_ENV: staging
 applications:
 - name: api
   path: .
   host: fec-prod-api
   env:
-    PRODUCTION: True
+    FEC_API_DEBUG: False
```

Comment:
If environment variables are cast to strings, this will set `FEC_API_DEBUG` to `"false"`, which is truthy.